### DISCUSSION TITLE
YetiForce 7.0.2 release

### DISCUSSION THREAD

**Discussion by paula-w** [YetiForceCompany/YetiForce#29] [Releases]
Hello everyone!

**We’re happy to announce that YetiForce 7.0.2 has been released.**

Below, you can find all the necessary links to download the latest stable version, test it, and find out what’s new.

- [DOWNLOAD](https://api.yetiforce.eu/download/crm/github/7.0.2-complete)
- [DEMO](https://demo.yetiforce.com/)
- [LICENSE](https://yetiforce.com/en/license.html)

**Comment by Ilamparithi-Tamizhanangu** [comment]
Hi, 

When i try to install the YF 7.0.0 / 7.0.1 / 7.0.2 using complete installation files, i am getting the error as Installation unsuccessful.

Please find the below screenshot. 
![YF7 installation error](https://github.com/user-attachments/assets/741946ab-0b56-4e70-bc71-231df21ed3c3)

when i install the YF 6.4, the installation was successfull and i updated it to the version 7.0.1. 

Could you please guide me to solve this error.

Thanks & regards,
Ilamparithi.

**Comment by gmisiolek-sbm** [comment]
Where can we see changes log? Because on your website is only 7.0.1: https://doc.yetiforce.com/introduction/release-notes-changelog/
